Fix Grammar

Bilibox opened the pull request:
Line 2 - Run on sentence, restructure to fix
Line 16 - fix improper sentence structure. 
Line 19 - the is required for an object (Black Market Shop)
Line 84 - Require would be more correct than make 
Line 86 - Add missing comma
Line 87 - remove s from separate(doesnt need an s), Member should be plural
Line 89 - Registered shouldn't be capitalized since the tag isn't, add a space to usergroup to keep previous formatting
Line 91 - fix improper sentence, fix capitalization
Line 94 - too much stuff is informal - fix to be more formal - needs to be reviewed more to see what better words can be used.
Line 101 - Registered shouldn't be capitalized since the tag isn't, add a space to usergroup to keep previous formatting
Line 109 - add period to keep formatting
Line 111 - Add "the" for the object 4k/packs
Line 122 - shouldn't be fully capitalized as the tag itself isn't. 
Line 124 - shouldn't be fully capitalized as the tag itself isn't. 
Line 129 - remove uppercase letters to keep the formatting used later
Line 155 - remove for - improper grammar, fix 5$ 
Line 158 - add The for object helpdesk
Line 160 - keep formatting
Line 179 - Fix question to be a proper sentence
Line 182 - Fix question to be a proper sentence
Line 183 - Fix structure of the sentence to make more sense
Line 186 - Fix question to be a proper sentence
Line 187 - add the for both objects
Line 188 - Create space between the 2 Q&A's 
Line 189 - Fix run on sentence. End original statement and start the question
Line 190 - Fix incorrect verbiage. - Should be looked at again to see if it can be fixed more
Line 191 - add a - PM is singular
Line 194 - Add comma & period to avoid run on sentence. Keep phrasing the same.
Line 197 - in the group not under the group. - Should look into changing the end to mention 4k/packs section instead.
Line 198 - Fix phrasing, add comma to avoid run on sentence. add the for object pirate tag. end sentence start sentence with an article keep phrasing the same throughout the line. add the for object shop. for is the correct usage instead of with. 
Line 200 - Singular item. Fix question to be a proper sentence
Line 201 - Singular item(Temp password/admin). semicolon would be the proper use here. fix wording to explain better. 
Line 203 - Fix question to be a proper sentence
Line 204 - remove do(Breaks sentence) fix wording for proper sentence structure
Line 207 - Keep formatting the same as previous usage.
Line 210 - fix incorrect wording.
Line 213 - Fix spelling error. add the for object img tag.
Line 215 - incorrect usage of a comma, object image hoster requires an. add comma to avoid run on sentence
Line 217 - remove unneeded words. End sentence to avoid run on sentence. x2 rephrase ending sentence. add the for object preview/edit mode
Line 219 - Fix question to be a proper sentence
Line 222 - Fix question to be a proper sentence
Line 223 - fix run on sentence, Shouldn't start a sentence with a contraction.
Line 225 - Fix question to be a proper sentence
Line 226 - add comma for defining clause. fix grammar
Line 228 - Fix question to be a proper sentence
Line 229 - Fix capitalization to keep previous formatting. add the for object golden guides. fix guide to be plural. add link to Golden Guides text. make post plural. keep consistent formatting.
Line 232 - keep consistent formatting throughout FAQ. fix wording so sentence makes sense. remove unneeded capitalization. replace in with every
Line 234 - should be with not for. (correct wording)
Line 235 - add with,  conjoin sentences to make it flow. 
Line 237 - fix working to be first person. (following formatting). capitalize Account to keep formatting with "Post" in next line. Fix certain wording to correct the sentence structure.
Line 241 - fix the question to fit the answer.
Line 242 - rearrange sentence for it to make more sense. 
Line 244 - Keep first person formatting.
Line 245 - Correct formatting
Line 248 - change a to the capitalize Pirate 
Line 250 - Fix question to be a proper sentence
Line 251 - Keep same formatting of labeling the section name.
Line 253 - improper usage of plural for accounts. Capitalize Gsuite (Object) change buy with purchase(use same wording as the shop)
Line 254 -  fix sentence to not be so weird. use same wording as the shop
Line 256 - Change buy with purchase to use the same wording from the shop.
Line 257 - attempt to fix sentence structure. It's mostly a run on sentence and should be looked at in the future.
Line 259 - for not of. 
Line 260 - combine sentences. fix structure.
Line 263 - Fix wording and combine sentence.
Line 265 - Which not what (lists)
Line 274 - keep first person sentence structure.
Line 275 - singular GD & Mega link, needs a before downcloud tag
Line 279/280 - Move ln 280 up to 279 as they talk about the same thing. replace are with is.
Line 281 - add the for object.
Line 283 - add section name to keep sentence structure from previous formatting.
Line 285 - fix sentence to better explain and make more sense.

Commit:
Fix Grammar

Line 2 - Run on sentence, restructure to fix
Line 16 - fix improper sentence structure. 
Line 19 - the is required for an object (Black Market Shop)
Line 84 - Require would be more correct than make 
Line 86 - Add missing comma
Line 87 - remove s from separate(doesnt need an s), Member should be plural
Line 89 - Registered shouldn't be capitalized since the tag isn't, add a space to usergroup to keep previous formatting
Line 91 - fix improper sentence, fix capitalization
Line 94 - too much stuff is informal - fix to be more formal - needs to be reviewed more to see what better words can be used.
Line 101 - Registered shouldn't be capitalized since the tag isn't, add a space to usergroup to keep previous formatting
Line 109 - add period to keep formatting
Line 111 - Add "the" for the object 4k/packs
Line 122 - shouldn't be fully capitalized as the tag itself isn't. 
Line 124 - shouldn't be fully capitalized as the tag itself isn't. 
Line 129 - remove uppercase letters to keep the formatting used later
Line 155 - remove for - improper grammar, fix 5$ 
Line 158 - add The for object helpdesk
Line 160 - keep formatting
Line 179 - Fix question to be a proper sentence
Line 182 - Fix question to be a proper sentence
Line 183 - Fix structure of the sentence to make more sense
Line 186 - Fix question to be a proper sentence
Line 187 - add the for both objects
Line 188 - Create space between the 2 Q&A's 
Line 189 - Fix run on sentence. End original statement and start the question
Line 190 - Fix incorrect verbiage. - Should be looked at again to see if it can be fixed more
Line 191 - add a - PM is singular
Line 194 - Add comma & period to avoid run on sentence. Keep phrasing the same.
Line 197 - in the group not under the group. - Should look into changing the end to mention 4k/packs section instead.
Line 198 - Fix phrasing, add comma to avoid run on sentence. add the for object pirate tag. end sentence start sentence with an article keep phrasing the same throughout the line. add the for object shop. for is the correct usage instead of with. 
Line 200 - Singular item. Fix question to be a proper sentence
Line 201 - Singular item(Temp password/admin). semicolon would be the proper use here. fix wording to explain better. 
Line 203 - Fix question to be a proper sentence
Line 204 - remove do(Breaks sentence) fix wording for proper sentence structure
Line 207 - Keep formatting the same as previous usage.
Line 210 - fix incorrect wording.
Line 213 - Fix spelling error. add the for object img tag.
Line 215 - incorrect usage of a comma, object image hoster requires an. add comma to avoid run on sentence
Line 217 - remove unneeded words. End sentence to avoid run on sentence. x2 rephrase ending sentence. add the for object preview/edit mode
Line 219 - Fix question to be a proper sentence
Line 222 - Fix question to be a proper sentence
Line 223 - fix run on sentence, Shouldn't start a sentence with a contraction.
Line 225 - Fix question to be a proper sentence
Line 226 - add comma for defining clause. fix grammar
Line 228 - Fix question to be a proper sentence
Line 229 - Fix capitalization to keep previous formatting. add the for object golden guides. fix guide to be plural. add link to Golden Guides text. make post plural. keep consistent formatting.
Line 232 - keep consistent formatting throughout FAQ. fix wording so sentence makes sense. remove unneeded capitalization. replace in with every
Line 234 - should be with not for. (correct wording)
Line 235 - add with,  conjoin sentences to make it flow. 
Line 237 - fix working to be first person. (following formatting). capitalize Account to keep formatting with "Post" in next line. Fix certain wording to correct the sentence structure.
Line 241 - fix the question to fit the answer.
Line 242 - rearrange sentence for it to make more sense. 
Line 244 - Keep first person formatting.
Line 245 - Correct formatting
Line 248 - change a to the capitalize Pirate 
Line 250 - Fix question to be a proper sentence
Line 251 - Keep same formatting of labeling the section name.
Line 253 - improper usage of plural for accounts. Capitalize Gsuite (Object) change buy with purchase(use same wording as the shop)
Line 254 -  fix sentence to not be so weird. use same wording as the shop
Line 256 - Change buy with purchase to use the same wording from the shop.
Line 257 - attempt to fix sentence structure. It's mostly a run on sentence and should be looked at in the future.
Line 259 - for not of. 
Line 260 - combine sentences. fix structure.
Line 263 - Fix wording and combine sentence.
Line 265 - Which not what (lists)
Line 274 - keep first person sentence structure.
Line 275 - singular GD & Mega link, needs a before downcloud tag
Line 279/280 - Move ln 280 up to 279 as they talk about the same thing. replace are with is.
Line 281 - add the for object.
Line 283 - add section name to keep sentence structure from previous formatting.
Line 285 - fix sentence to better explain and make more sense.

Bilibox commented:
Note: lines 190/197/257 should be looked into after this PR to see how they can be further changed to make more sense.

Commit:
Fix Q: Which file hosters does the forum accept?

Fixes the sentence to make more sense. 
Which file hosters does the forum accept? or What file hosters does the forum accept? are both incorrect in a grammar sense. This makes the question more legible and flow better.

Commit:
Add Accorsi's fix to last question

Add's Accorsi's commit change to fix the last questions formatting.

Bilibox commented:
Added a grammar fix & @Accorsi66 's commit.

KurashiAOI commented on FAQ.md:
```diff
@@ -176,92 +176,93 @@ Post quality is judged by the amount of effort you put into the creation of your
 
 ### Q: How does one add a picture to a post? Also, what image hosters I can use?
-**A**: You can do this by clicking on the [little image icon](https://uploaddeimagens.com.br/images/002/359/857/original/2019-09-19_181337.png?1568927759) at the post tollbar or using the hotkey `Ctrl+P`. If you are using [bbcode](https://www.phpbb.com/), you have to use `[IMG]` tag. Example: `[IMG]https://hoster.com/image.png[/IMG]`.
+**A**: You can do this by clicking on the [little image icon](https://uploaddeimagens.com.br/images/002/359/857/original/2019-09-19_181337.png?1568927759) at the post toolbar or using the hotkey `Ctrl+P`. If you are using [bbcode](https://www.phpbb.com/), you have to use the `[IMG]` tag. Example: `[IMG]https://hoster.com/image.png[/IMG]`.
```
Maybe you can put  [bbcode] (https://blackpearl.biz/help/bb-codes/) instead of [bbcode] (https://www.phpbb.com/) . I think it's better for new user to read the already available reference from BlackPearl instead of other external sources.

KurashiAOI commented on FAQ.md:
```diff
@@ -176,92 +176,93 @@ Post quality is judged by the amount of effort you put into the creation of your
+**A**: You can do this by clicking on the [little image icon](https://uploaddeimagens.com.br/images/002/359/857/original/2019-09-19_181337.png?1568927759) at the post toolbar or using the hotkey `Ctrl+P`. If you are using [bbcode](https://www.phpbb.com/), you have to use the `[IMG]` tag. Example: `[IMG]https://hoster.com/image.png[/IMG]`.
 
-For that, you need to use image hoster. We recommend [ImgBB](https://imgbb.com/), [imgur](https://imgur.com) or [PostImages](https://postimages.org/) but you can use any other since it allows you to reach the original image link. Example: `https://hoster.com/image.png`.
+For that you need to use an image hoster. We recommend [ImgBB](https://imgbb.com/), [imgur](https://imgur.com) or [PostImages](https://postimages.org/) but you can use any other, since it allows you to reach the original image link. Example: `https://hoster.com/image.png`.
```
I think it is ''as long as it allows'' instead of ''since it allows''. Correct me if I'm wrong though.

Bilibox replied:
I believe both are technically correct but "as long as it allows" seems to flow better so I'll change that part.

KurashiAOI commented on FAQ.md:
```diff
@@ -176,92 +176,93 @@ Post quality is judged by the amount of effort you put into the creation of your
-### Q: How to remove my post from the forum? I can't see any option.
-**A**: First you need to click on the report button, this can be found at the bottom of the post. Then you need to give the report reason and click on the report. Alternatively, under the [Telegram channel](https://t.me/joinchat/KUVnAlQmTohLLwZEZcGv8A) drop your link and ask for a remove. In both ways, you have to wait until the staff process your report.
+### Q: How do I remove my post from the forum? I can't see any option.
+**A**: First you need to click on the report button, this can be found at the bottom of the post. Then you need to give the report reason and click on the report. Alternatively, under the [Telegram channel](https://t.me/joinchat/KUVnAlQmTohLLwZEZcGv8A), drop your link and ask for it to be remove. In both ways, you have to wait until the staff process your report.
```
''to be remove'' > ''to be removed''

KurashiAOI commented on FAQ.md:
```diff
@@ -176,92 +176,93 @@ Post quality is judged by the amount of effort you put into the creation of your
 
 ### Q: How can I change my user name?
-**A**: If you have a DONATOR Tag, then you can change your user name in every 24 hours. Moreover, If you do not have a DONATOR Tag, then you can change your user name only once in 30 days.
+**A**: If you have a Donator tag, then you can change your user name every 24 hours. Moreover, if you do not have a Donator tag, then you can change your user name only once every 30 days.
```
''Moreover,'' > ''However,''

KurashiAOI commented on FAQ.md:
```diff
@@ -176,92 +176,93 @@ Post quality is judged by the amount of effort you put into the creation of your
-### Q: what about alternative accounts?
-**A**: No alternate accounts are allowed. If found any, both accounts will be banned.
+### Q: Are alternate accounts allowed?
+**A**: No, alternate accounts are NOT allowed. If any are found, both accounts will be banned.
```
''are found'' > ''were found''

Bilibox replied:
For this I'm going to have to use "were to be found" to keep the future tense of the sentence intact. otherwise it would be mixing past tense and future tense in the same sentence. let me know if that's okay for you or if you think it should be something different.

KurashiAOI commented on FAQ.md:
```diff
@@ -270,16 +271,15 @@ In case you prefer to use Imgur, then instead of copying the default link that t
-To get the password you need to have 500 likes on your profile, after this please go to this [thread](https://blackpearl.biz/threads/907/).
+### Q: Why does the [How to get free Account on Various Service](https://blackpearl.biz/forums/116/) section have a password?
+**A**: This is a forum with tutorials to create profiles on various services. This section is locked behind a password to avoid leechers & copyright trolls.
+To get the password, you need to have 500 likes on your profile. Once you have that, please go to this [thread](https://blackpearl.biz/threads/907/) to recieve the password.
```
''recieve'' > ''receive''

Commit:
Correct additional error's

Line 4 - Fix "what to do now" to first person context to keep sentence structure throughout FAQ
Line 213 -KurashiAOI fix- change BBCode link to point to Black Pearl BBCode.(this is a MUCH better alternative than the phpbb site) Also changed bbcode to BBCode as that's how it should be written. 
Line 215 -KurashiAOI fix- ''as long as it allows'' instead of ''since it allows''. Sentence flow is smoother with this change.
Line 226 -KurashiAOI fix- Correct spelling error "remove" > "removed". Also added "will" to add future tense and make the sentence run smoother
Line 232 -KurashiAOI fix- "Moreover,'' > ''However,''
Line 242 -KurashiAOI fix- ''are found'' > ''were found'' adding to KurashiAOI's fix "were to be found" to keep future tense. Changing "will" to "would"
Line 285 -KurashiAOI fix- fix spelling error ''recieve'' > ''receive''
